demo/src/commonMain/kotlin/Arithmetic.kt

```diff
@@ -1,6 +1,6 @@
-@file:Suppress("MemberVisibilityCanBePrivate")
+@file:Suppress("MemberVisibilityCanBePrivate", "PackageDirectoryMismatch")
 
-package me.alllex.parsus.demo
+package me.alllex.parsus.demo.arithmetic
```

Comment:
Kotlin/Native can't handle multiple `fun main()`s in the same package - so I made this quick tweak